packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -164,6 +167,33 @@ const Actions = {
     };
     graph.backgroundColor(model.style.background);
   },
+  evaluate: () => {
```

Comment:
I am not sure I understand what this function does and why it's called `evaluate`, can you help me understand?

Reply:
It checks the values of the gui controllers (only one currently, but there will be more), modifies the graph data accordingly, and then updates the graph. Would it be better if I called it something like `evaluateControllers`?

Reply:
That makes more sense. I'm going to try again, but I don't know if I'm fully understanding how this should be structured just yet.